sdm.R

```diff
@@ -0,0 +1,95 @@
+#!/apps/R/3.2.0/bin/Rscript
+
+devtools::load_all()
+library(parallel)
+library(tidyverse)
+library(gbm)
+
+dir.create("sdm_predictions", showWarnings = FALSE)
+
+start_yr = 1982
+end_yr = 2013
+min_num_yrs = 25
+last_train_year = 2003
+
+# GBM parameters
+first_validation_year = 2000 # for temporal cross-validation
+interaction.depth = 8
+max_n_trees = 2E4
```

Comment:
20,000 trees is probably a bit much. I'd be willing to bet the accuracy improvements after ~2000 trees are probably < 0.01.  Cutting this down would cut down the processing time by a bunch.